src/main/java/com/epam/emotionalhelp/controller/response/ResponseHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.emotionalhelp.controller.response;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ResponseHandler {
+    public static ResponseEntity<Object> generateResponse(String message, HttpStatus status, Object responseObj) {
```

Comment:
Can we consider annotation-based configuration for response status code?

Resource: [Using Spring @ResponseStatus to Set HTTP Status Code](https://www.baeldung.com/spring-response-status)